modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceSettings.testcase

```diff
@@ -101,4 +103,15 @@ definition {
 		AssertElementPresent(locator1 = "Message#SUCCESS");
 	}
 
+	test ViewInstanceCurrenciesAvailable {
+		property environment.acceptance = "true";
+		property test.name.skip.portal.instance = "CommerceSmoke#ViewAvailableCurrencies";
+
+		CommerceNavigator.gotoPortlet(
+			category = "Store Management",
+			portlet = "Currencies");
+
+		CommerceEntry.viewCurrenciesEntry(currenciesList = "US Dollar,Australian Dollar,British Pound,Canadian Dollar,Chinese Yuan Renminbi,Euro,Hong Kong Dollar,Japanese Yen,Indian Rupee,Brazilian real");
```

Comment:
@iovaneale should `Brazilian real` be `Brazilian Real`?

Reply:
@dsitu I had the same exact doubt, but it seems that that's right with the lowercase "r". See https://en.wikipedia.org/wiki/Brazilian_real

Reply:
@iovaneale, ahh ic, thanks for the clarification. I wasn't sure if this was suppose to be the same as our language key, https://github.com/liferay/liferay-portal/blob/master/portal-impl/src/content/Language.properties#L6292. But if it is unrelated it should be fine.

Reply:
@dsitu At this point, I am not sure it is unrelated. Anyway, it seems there is an inconsistency between the lang key and what's actually shown (Brazilian real). I will ask Corbin and then eventually ask for it to be merged.

Reply:
@dsitu Another source (https://www.countries-ofthe-world.com/world-currencies.html) says the same and it's not the only case. If you agree we can merge this PR and in case adapt it later.

Reply:
@iovaneale tanks for looking into it, yeah I think its good.

Reply:
:+1:

Reply:
Hey @dsitu, @iovaneale: it should follow the same pattern as the other currencies. If it were a sentence, you're correct that "real" would be all lowercase, but the same logic would then apply to all the currencies in the list (e.g. "US dollar"). The crux is that we should be following list rules. In a list with short list items like we do in this case, American English will usually capitalize the first letter for aesthetic purposes. We should follow the same pattern across the board and go with `Brazilian Real`, not `Brazilian real`. Thanks!